### ISSUE TITLE
Backlog: support memory limits / reservations in YAML

### ISSUE THREAD

**Issue by alexellis** [openfaas/faas-cli#223]
## Description

Backlog: support memory limits / reservations in YAML

## Acceptance criteria

* Integrates with https://github.com/openfaas/faas/pull/385 tested through Docker Swarm
* Includes following schema elements at per-function level:

At function level:

```
        // Limits for function
	Limits *FunctionResources `json:"limits"`

	// Requests of resources requested by function
	Requests *FunctionResources `json:"requests"`
```

```
// FunctionResources Memory and CPU
type FunctionResources struct {
	Memory string `json:"memory"`
	CPU    string `json:"cpu"`
}
```

* Console output from test-plan should be shown along with PR

**Comment by alexellis** [comment]
Fixed and released

**Comment by ericstoekl** [comment]
Derek close: fixed in #225

**Comment by alexellis** [comment]
Thanks for closing @ericstoekl
